shortcuts.js

```diff
@@ -0,0 +1,17 @@
+import { button } from './index.js';
+
+export const buttons = {
+    'button': button.buttonSecondary,
+    'button--secondary': button.buttonSecondary,
+    'button--default': button.buttonSecondary,
```

Comment:
Should they all be secondary? 😅

Reply:
https://github.com/fabric-ds/css/blob/next/src/components/button.css#L3 yes